frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -16,28 +36,33 @@ const ModuleUnionCard: FC<ModuleUnionCardProps> = ({
   // ModuleUnionCard is a generic component for rendering cards of any
   // model type, removing the need to import every card component.
   let content;
+  const highlightSnippet =
+    module?.meta?.highlight?.text?.[0] ?? module?.meta?.highlight?.description?.[0]; // highlight text or highlight description
```

Comment:
I'm not sure exactly how the highlight API works, but we use `text` here because that field exists on quotes, and `description` because that field exists on entities, right? If so, do we need `elaboration` for propositions/occurrences? And I'm not sure what the fields would be for images and sources, if we want those highlighted as well.

Reply:
In `apps/search/api/filters/elastic_filters.py`, I found this code:
```
        # TODO: refactor & improve this. currently only applying highlights to quote#text and occurrence#description
        qs = qs.highlight(
            'text',
            number_of_fragments=1,
            type='plain',
            pre_tags=['<mark>'],
            post_tags=['</mark>'],
        )
        qs = qs.highlight(
            'description',
            number_of_fragments=1,
            type='plain',
            pre_tags=['<mark>'],
            post_tags=['</mark>'],
        )
```

For now, we could just just add `elaboration` to this like so...
```
        qs = qs.highlight(
            'text',
            number_of_fragments=1,
            type='plain',
            pre_tags=['<mark>'],
            post_tags=['</mark>'],
        ).highlight(
            'description',
            number_of_fragments=1,
            type='plain',
            pre_tags=['<mark>'],
            post_tags=['</mark>'],
        ).highlight(
            'elaboration',
            number_of_fragments=1,
            type='plain',
            pre_tags=['<mark>'],
            post_tags=['</mark>'],
        )
```